src/theme/theme.js

```diff
@@ -1,12 +1,615 @@
+/* eslint-disable no-underscore-dangle */
 // The default theme.js for Paragon. The structure of this object follows the
 // theme specification found here: https://system-ui.com/theme/. We prefer to add
 // to this theme over time rather than all valid keys at once to ensure everything
 // present here is used in the system.
```

Comment:
What exists below is a departure from the strategy articulated in this comment. And what's below doesn't follow the theme specification from system-ui. Let's discuss if this direct port of Bootstrap is desirable and if so, what next steps should be.